### PR TITLE
Incrementing the ammount of unit tests

### PR DESCRIPTION
**Pull request by osmarpetry**
**- What I did**
Incremented more tests to the project

**- How I did it**
I started to create tests in the`--watch` mode, just increment some of then, not all tests I want

**- How to verify it**
Read the code and run all the unit tests 

**- Description for the changelog**

More unit tests to Angular

### CODE REVIEW

**Comment by Tiagoperes** [comment]
Thank you for your contribution Osmar. I'll review it as soon as possible!

**Comment by osmarpetry** [comment]
> Thank you for your contribution Osmar. I'll review it as soon as possible!

I did the same on inputs to check `readonly` and `disabled` from `beagle-web-react`. If don't want it, I can easily remove it here. Just did it on JS, because HTML can't prevent the value to change

**Comment by Tiagoperes** [comment]
Hi Osmar, sorry for the long delays, we're quite busy here.

Since these tests are analogous to the ones you implemented for React, I'll wait for thos to be approved before checking these. Ok?

File: src/__tests__/components/beagle-button.spec.ts [hunk 1]
```diff
@@ -60,4 +60,22 @@ describe('BeagleButtonComponent', () => {
...
+  it('should trigger handleClick function', () => {
+    component.onPress = () => null
+    fixture.detectChanges();
+    spyOn(component, 'onPress');
+    let button = fixture.debugElement.nativeElement.querySelector('button');
+    button.click()
+    expect(component.onPress).toHaveBeenCalled();
+  })
+
+  it('should trigger handleClick function when is disabled', () => {
```
Comment (Tiagoperes) [inline src/__tests__/components/beagle-button.spec.ts:72]: should **not** trigger

File: src/components/beagle-button/beagle-button.component.ts [hunk 2]
```diff
@@ -54,11 +54,12 @@ export class BeagleButtonComponent extends BeagleComponent
   }
 
   handleClick() {
+    this.onPress && this.type === 'button' && !this.disabled && this.onPress()
```
Comment (Tiagoperes) [inline src/components/beagle-button/beagle-button.component.ts:57]: For the same reason explained in the react PR, the logic here was correct. There's no need to add this verification for the attribute `disabled`.

We should not make a logic in the source code more complex just to suit a test.

As I said before, to test for a disabled button you should just check if the final button element in the HTML received the attribute "disabled". If it did, we know the browser works, so it won't call the handler once pressed. We must trust the browser is working, we should test only our application, never the browser.

**Review by Tiagoperes — CHANGES_REQUESTED** [review]

File: src/components/beagle-button/beagle-button.component.ts (hunk 2 above)
Comment (osmarpetry) [reply src/components/beagle-button/beagle-button.component.ts:57]: I thought I have removed this alteration from the PR. I will check it, in home on the computer

**Comment by osmarpetry** [comment]
@Tiagoperes  I did the required changes

**Review by Tiagoperes — DISMISSED** [review]

**Review by Tiagoperes — APPROVED** [review]

**Review by carolinegoncalveszup — APPROVED** [review]

**Pull request merged**